triveni-codev/GBWebApp-dev
# Justin_Outputs-Section_Fix-test-connection_1203813250165127

ASANA ticket: https://app.asana.com/0/1203786645839927/1203813250165127/f

**Description:** Currently, there's an undefined error upon testing the connection of Output's IP address. Also, we need to amend the connection result message based on the API response.
- created a constant variable which will identify if the test connection data is undefined or not.
- amended connection result message (string) based on the API response.

**TO TEST:**
- create/update an Output (triveni-carousel type)
- enter **10.77.35.18** in IP address and **5001** for the IP port ~ see reference here: https://docs.google.com/spreadsheets/d/1REazGVN4QMl-STe1KWBRfPskvKpZ8uqf0iECTGz8Rj8/edit#gid=2014828454
- test connection
- counter check the test case in the clientapp

cc: @triveni-codev
